src/pages/GalleryDetails/components/ImageDialog.tsx

```diff
@@ -0,0 +1,82 @@
+import { Dialog as MuiDialog, DialogProps as MuiDialogProps, Paper, Typography } from '@mui/material';
+import { makeStyles } from 'makeStyles';
+
+import { PermissionApp } from 'types/Enums';
+
+import { HavePermission } from 'hooks/User';
+
+import PictureEditorDialog from './PictureEditor';
+
+export type ImageTextProps = {
+  title: string;
+  description: string;
+};
+
+const useStyles = makeStyles()((theme) => ({
+  background: {
+    position: 'absolute',
+    left: 0,
+    bottom: 0,
+    zIndex: 1000,
+    background: 'rgba(255, 255, 255, 0.5)',
+    p: '5px',
+    color: 'white',
+    maxWidth: '100%',
+    width: '100%',
+    maxHeight: '20%',
+    height: '100%',
+  },
+  text: {
+    float: 'left',
+    position: 'absolute',
+    left: theme.spacing(1),
+    top: theme.spacing(1),
+    zIndex: 1000,
+    color: 'black',
+  },
+}));
+
+const ImageText = ({ title, description }: ImageTextProps) => {
+  const { classes } = useStyles();
+  return (
+    <div className={classes.background}>
+      <div className={classes.text}>
+        <Typography fontSize={'150%'} variant='h2'>
+          {title}
+        </Typography>
+        <Typography fontSize={'100%'} variant='h4'>
+          {description}
+        </Typography>
+      </div>
+    </div>
+  );
+};
```

Comment:
Trenger du en egen komponent for dette? Legg det heller rett inn der den brukes. `fontSize=100%` ser også litt rart ut.